src/window/intro_video.c

```diff
@@ -4,22 +4,37 @@
 #include "graphics/screen.h"
 #include "graphics/video.h"
 #include "graphics/window.h"
+#include "sound/music.h"
+
+#define NUM_INTRO_VIDEOS 2
 
 static struct {
     int width;
     int height;
+    int current_video;
 } data;
 
-static int init(void)
+static const char *intro_videos[NUM_INTRO_VIDEOS] = { "smk/logo.smk", "smk/intro.smk" };
```

Comment:
Also add `smk/credits.smk` (and increase NUM_VIDEOS to 3)

Reply:
Right now the videos are playing in succession. If I do that, credits will play right after the intro videos. Are you sure about that?

Reply:
Yes. That's what C3 also does.